docs/source/usage/tbg.rst

```diff
@@ -21,7 +21,8 @@ Also, ``.tpl`` files takes care of how to spawn a process when scheduled, e.g. w
 After combining the *machine independent* (portable) ``.cfg`` file from user input with the *machine dependent* ``.tpl`` file, ``tbg`` can submit the requested job to the batch system.
 
 Last but not least, one usually wants to store the input of a simulation with its output.
-``tbg`` conveniently automates this task before submission.
+``tbg`` conveniently automates this task before submission. The ``.tpl`` and the ``.cfg`` files that were used to start the simulation can be found in ``<tbg destination dir>/tbg/`` and
+can be used together with the ``.param`` files from ``<tbg destination dir>/input/.../param/`` to recreate the simulation setup.
```

Comment:
Maybe even remove the newline since all the wrapping is done automatically, anyway.

Reply:
Yes, we normally start each sentence from a new line in `.rst` files since it does not affect how the docs look eventually but makes it easier to edit raw files.